pkgs/servers/sql/postgresql/ext/omnigres.nix

```diff
@@ -0,0 +1,134 @@
+{
+  buildEnv,
+  bison,
+  boost,
+  brotli,
+  bzip2,
+  cmake,
+  doxygen,
+  fetchFromGitHub,
+  flex,
+  git,
+  lib,
+  libcap,
+  libtool,
+  libuv,
+  libxml2,
+  lld,
+  netcat,
+  openssl,
+  perl,
+  pkg-config,
+  postgresql,
+  postgresqlBuildExtension,
+  postgresqlTestExtension,
+  postgresqlPackages,
+  python3,
+  python3Packages,
+  readline,
+  stdenv,
+  zlib,
+}:
+
+let
+  # TODO: I'll remove this as well, just testing stuff
+  extensions = buildEnv {
+    name = "postgresql-custom";
+    paths = [
+      postgresql
+      postgresqlPackages.plpython3
+    ];
+  };
+in
+postgresqlBuildExtension (finalAttrs: {
+  pname = "omnigres";
+  version = "413feff21f9f7310023d8cfd92b83f2a251b1aa4";
+
+  src = fetchFromGitHub {
+    owner = "omnigres";
+    repo = "omnigres";
+    rev = finalAttrs.version;
+    hash = "sha256-OEKXz/98VpaBhLhC2mkWx73lQmlflv3sI7eXLvgoDiI=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    flex
+    libtool
+    pkg-config
+    perl
+    python3
+  ];
+  buildInputs = postgresql.buildInputs ++ [
+    bison
+    boost
+    brotli
+    bzip2
+    doxygen
+    flex
+    git
+    libcap
+    libuv
+    libxml2
+    lld
+    netcat
+    openssl
+    perl
+    python3
+    python3Packages.build
+    readline
+    zlib
+  ];
+
+  cmakeFlags = [
+    "-DNETCAT=${netcat}/bin/nc"
+    "-DOPENSSL_CONFIGURED=1"
+    "-DPG_CONFIG=${postgresql.pg_config}/bin/pg_config"
+    # Can remove this later after hack is deprecated
+    "-DPostgreSQL_EXTENSION_DIR=${lib.getDev extensions}/share/postgresql/extension"
```

Comment:
@wolfgangwalther and this is where the extensions are fetch from. Setting the extension dir like this will make it break during the install phase.